ironmon_tracker/Program.lua

```diff
@@ -131,6 +138,18 @@ function Program.update()
 					Tracker.Data.hasCheckedSummary = true
 				end
 			end
+
+			if Options["Display repel usage"] and not Battle.inBattle then
+				-- Check if the player is in the start menu (for hiding the repel usage icon)
+				-- Don't have a good solution for Ruby/Sapphire yet, so skip checking for that game
+				if GameSettings.game ~= 1 then
```

Comment:
Moved this check to within the function, though to note if someone else wants to use the start menu check for something else i'd recommend moving the line calling `Program.isInStartMenu` itself somewhere more general as it sets a global `Program.isInStartMenu` variable, and then using that global variable wherever they want to have logic for in/out of start menu.

It's here currently since it's only used for the repel stuff atm, just to reduce unnecessary calls to it.